packages/ui/package.json

```diff
@@ -11,6 +11,10 @@
     },
     "./*": "./*"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vitest-dev/vitest.git"
```

Comment:
Add `directory` property like in `vite-node`
https://github.com/vitest-dev/vitest/blob/b9d092b5fb098ec9836d8e8ca9088c042f945457/packages/vite-node/package.json#L12

Reply:
updated, thanks!